doc/schema_template.md

```diff
@@ -36,11 +51,12 @@ $tag:tool|inputs|param|sanitizer|valid|remove://complexType[@name='SanitizerVali
 $tag:tool|inputs|param|sanitizer|mapping://complexType[@name='SanitizerMapping']
 $tag:tool|inputs|param|sanitizer|mapping|add://complexType[@name='SanitizerMappingAdd']
 $tag:tool|inputs|param|sanitizer|mapping|remove://complexType[@name='SanitizerMappingRemove']
-$tag:tool|configfiles://complexType[@name='ConfigFiles']
-$tag:tool|configfiles|configfile://complexType[@name='ConfigFile']
-$tag:tool|configfiles|inputs://complexType[@name='ConfigInputs']
-$tag:tool|environment_variables://complexType[@name='EnvironmentVariables']
-$tag:tool|environment_variables|environment_variable://complexType[@name='EnvironmentVariable']
+$tag:tool|request_param_translation://complexType[@name='RequestParameterTranslation']
```

Comment:
I'm kind of -0 on indexing these elements in this way. I didn't index them intentionally - but perhaps that was the wrong choice on my part.

Reply:
Is there any other way?
Also, if someone has to write a `data_source` tool, they need this info.